steps/tslint/0.0.2/step.yml

```diff
@@ -0,0 +1,49 @@
+title: Run tslint command
+summary: Runs `tslint` with the given args.
+description: Runs `tslint` with the given args.
+website: https://github.com/ibratoev/steps-tslint
+source_code_url: https://github.com/ibratoev/steps-tslint
+support_url: https://github.com/ibratoev/steps-tslint/issues
+published_at: 2017-03-22T16:03:54.333538005+02:00
+source:
+  git: https://github.com/ibratoev/steps-tslint.git
+  commit: 764d22569364b38e572a1f0c8783e26beca79aee
+host_os_tags:
+- osx
+- ubuntu
+project_type_tags:
+- ios
+- android
+type_tags:
```

Comment:
following type_tags are available on bitrise:

- access control
- artifact info
- installer
- deploy
- utility
- dependency
- code sign
- build
- test
- notification

could you please select one (or more) from the list?

_if you think non item of the list fits to this step, please let me know_